pkg/controller/argocd/hooks_test.go

```diff
@@ -29,6 +29,14 @@ func testClusterRoleHook(cr *argoprojv1alpha1.ArgoCD, v interface{}) error {
 	return nil
 }
 
+func testRoleBindingHook(cr *argoprojv1alpha1.ArgoCD, v interface{}) error {
+	switch o := v.(type) {
+	case *v1.RoleBinding:
+		o.RoleRef.Name = "admin"
```

Comment:
I'd tend to make this more obvious that it's _not_ the default value.

e.g. something like `just-a-test-admin` way it's clearer in the _test_ that it's definitely applying the change.

Reply:
updated it to `test-admin-role` !!